cdn_lambda/functions/map_to_s3/map_to_s3.py

```diff
@@ -8,7 +8,7 @@
 
 
 class LambdaClient(object):
-    def __init__(self, conf_file="map_to_s3.json"):
+    def __init__(self, conf_file="lambda_conf.json"):
```

Comment:
This is slightly different from the filename used in the repo, is that intentional?

Reply:
Nice catch! That was unintentional. Fixed.